src/redux/reducers/componentConfig/plotDataLoading.js

```diff
@@ -0,0 +1,16 @@
+import { initialPlotDataState } from './initialState';
+
+const plotDataLoading = (state, action) => {
```

Comment:
why are we storing this? to show some placeholder while we load? what did we do before that?

Reply:
Yes, to show placeholder while the plot is being loaded. Before this, we are checking for `!plotData.length` to show that the item is loading. I think having this `loading` state is much better, because we might have cases where plot data is empty (there actually is no data).